build.sbt

```diff
@@ -58,15 +66,24 @@ lazy val core = crossProject(JSPlatform, JVMPlatform)
       compilerPlugin("com.olegpy" %% "better-monadic-for" % "0.3.1")
     )
   )
-  .jvmSettings(
-    fork in Test := true,
-    fork in run := true
-  )
+  .jvmSettings(fork in Test := true, fork in run := true)
 lazy val coreJVM = core.jvm
 lazy val coreJS = core.js.settings(
   libraryDependencies += "io.github.cquiroz" %%% "scala-java-time" % "2.0.0-RC3" % Test
 )
 
+lazy val catsInterop = crossProject(JSPlatform, JVMPlatform)
```

Comment:
This might actually solve the build failure in the CI. By default it expects common files to be in a "shared" folder, but using `CrossType.Pure` it just expects the regular `src/main/scala` folder.